src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -9,7 +9,7 @@
 #include "IReadFile.h"
 #include "os.h"
 #include "irr/asset/format/convertColor.h"
-#include "irr/asset/ICPUImageView.h"
+#include "irr/asset/ICPUImage.h"
 
 namespace irr
```

Comment:
you could do all the required conversions first (or flatten) then an optional flip, this loader is a bit of a mess , i see two stages which can each do a convert.

Reply:
yeah it's a mess